packages/amplication-client/src/Resource/git/AuthResourceWithGithubCallback.tsx

```diff
@@ -13,6 +14,11 @@ const AuthResourceWithGithubCallback = () => {
       // close the popup
       window.close();
     },
+    refetchQueries: [
+      {
+        query: GET_PROJECTS,
+      },
+    ],
```

Comment:
Why do we need it?

Reply:
when the user adds a new git organization to their workspace, on the server side we remove the settings of the demo repo if it exits, so we want the lisf of projects to be refreshed because this is where we have the indication about the repo demo.

project.repoDemo may be true before the transaction, when become false after the user adds a new git organization